pylint/lint/pylinter.py

```diff
@@ -970,7 +969,7 @@ def check(self, files_or_modules):
                 files_or_modules,
             )
 
-    def check_single_file(self, name, filepath, modname):
+    def check_single_file(self, name: str, filepath: str, modname: str) -> None:
```

Comment:
These arguments seems to represent a `FileItem`.

Reply:
Needed some refactoring, but added!